src/components/Checkout/OrderTotals.vue

```diff
@@ -103,7 +103,18 @@
 			<matched-loan-kiva-credit :open-lightbox="openLightbox" />
 		</div>
 
-		<div class="order-total" data-testid="total-due">
+		<!-- variant e on donate item experiment -->
+		<donation-item
+			v-if="donateItemExperimentVersion === 'e' && Number(totals.loanReservationTotal) > 0 "
+			data-testid="basket-donation"
+			:donation="donationObject"
+			:order-total-variant="donateItemExperimentVersion === 'e'"
+			:loan-count="Number(totals.loanReservationTotal) > 0 ? 1 : 0"
+			:loan-reservation-total="Number(totals.loanReservationTotal)"
+			@refreshtotals="$emit('refreshtotals')"
+			@updating-totals="$emit('updating-totals', $event)"
+		/>
+		<div class="order-total tw-text-left md:tw-text-right" data-testid="total-due">
```

Comment:
Added the tailwind classes `tw-text-left md:tw-text-right` for 2 divs on this page, and removed the legacy breakpoint which do not align with our tailwinds breakpoint